test/unit/model/include.test.js

```diff
@@ -179,7 +179,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           include: [{ model: this.Project }],
         });
 
-        expect(options.include[0]).to.have.property('where').which.deep.equals({ active: true });
+        expect(options.include[0]).to.have.property('where').which.deep.equals({ active: true  });
```

Comment:
```suggestion
        expect(options.include[0]).to.have.property('where').which.deep.equals({ active: true });
```

not sure why the linter didn't catch this
will merge as soon as this suggestion is committed :)

Reply:
Don't know either 🤔

Reply:
I've removed the extra space, so it is good for the merge ;)